daffodil-cli/build.sbt

```diff
@@ -36,6 +36,22 @@ Universal / mappings ++= Seq(
   baseDirectory.value / "README.md" -> "README.md",
 )
 
+// When the sbt-native-packger plugin creates a tar/zip/RPM/etc, it does not include the
+// crossVersion in mapping of jars built from this project (i.e. org.apache.daffodil jars only).
+// There are cases where this crossVersion would be useful (e.g. consistenncy with Ivy jar names
```

Comment:
*consistency

Reply:
Will fix.

> I wonder if we should file an issue with the sbt-native-packager maintainer

sbt-native-packager unfortunately seems pretty dead. PR's aren't getting merged and releases aren't being made (even though some tags have been created). I think whoever the maintainers are have lost interest/time.